core/domain/rte_component_registry_test.py

```diff
@@ -34,35 +34,46 @@
 from core.domain import rte_component_registry
 from core.tests import test_utils
 
+from typing import List, Tuple, Type
+from typing_extensions import Final
+
 # File names ending in any of these suffixes will be ignored when checking for
 # RTE component validity.
-IGNORED_FILE_SUFFIXES = ['.pyc', '.DS_Store']
-RTE_THUMBNAIL_HEIGHT_PX = 16
-RTE_THUMBNAIL_WIDTH_PX = 16
+IGNORED_FILE_SUFFIXES: Final = ['.pyc', '.DS_Store']
+RTE_THUMBNAIL_HEIGHT_PX: Final = 16
+RTE_THUMBNAIL_WIDTH_PX: Final = 16
 
-_COMPONENT_CONFIG_SCHEMA = [
+# Here, we used object because every type is inherited from object class.
+_COMPONENT_CONFIG_SCHEMA: List[Tuple[str, Type[object]]] = [
```

Comment:
I think in the current setup (we use `from __future__ import annotations`) list should work.

Reply:
Not working i tried, MyPy throwing error:- `core/domain/rte_component_registry_test.py:47: error: Implicit generic "Any". Use "typing.List" and specify generic parameters  [type-arg]`

Reply:
Hmm, weird, can you try to dig into this more?

Reply:
I also tried to do the Typevar approach here but again mypy shows the same error. 

The suspected reason for this:
while declaring list type MyPy always expecting a generic parameter. that's why while declaring list type list[str] and List[str] are ok but  alone list is throwing error ( Implicit generic "Any". ). One way to suppress the error is by adding `--disable-implicit-any` flag in MyPy but I'll against that approach because it is applied to the whole codebase.

Reply:
Okay.